packages/components/package.json

```diff
@@ -77,7 +77,7 @@
     "ember-prism": "^0.12.0",
     "ember-qunit": "^5.1.5",
     "ember-resolver": "^8.0.3",
-    "ember-source": "~4.7.0",
+    "ember-source": "~4.8.1",
```

Comment:
@MelSumner for consistency would you be able to make this same change across the other 3 packages that use ember? I found that reduced the risk of yarn doing something surprising trying to resolve which version to use.